DATASTD-2454 Create New Sample Data For The New Identity Entities

mikeholm-simpat opened the pull request:
Added new sample data to the following XML files:

- Candidate
- Contact
- EducationOrganization
- StaffAssociation
- Student

mikeholm-simpat committed:
DATASTD-2454

mikeholm-simpat committed:
DATASTD-2454

stevenarnold-ed-fi commented on Samples/Sample XML/Candidate.xml:
```diff
@@ -379652,6 +379652,81 @@
+			</EducationOrganizationIdentity>
+		</EducationOrganizationReference>
+	<IdentificationCode>837381631</IdentificationCode>
+	<AssigningOrganizationIdentificationCode>SCOL_255901001</AssigningOrganizationIdentificationCode>
```
This pertains to all Identities:

AssigningOrganizationIdentificationCode is meant to be a more descriptive explanation of what type of code it is.  So the CandidateIdentificationSystemDescriptor might be a value of `local` and the AssigningOrganizationIdentificationCode could be `College of Ed id code` 

Descriptor could be `SSN`, AssigningOrganizationIdentificationCode might be `Social Security Number`

This does bring up a point I've thought before, we don't have a good enough description on what this field is meant to represent and is confusing, will take note to discuss updating the field description.

Additionally, AssigningOrganizationIdentificationCode is optional, you should have at least 1 entry in each identity type where this doesn't have a value

mikeholm-simpat replied:
Updated changes based on direction above and re-submitted.

stevenarnold-ed-fi reviewed (changes requested):
This is pretty close to what were looking for, I left a comment on the candidate version that applies to all the different types with a couple of updates

mikeholm-simpat committed:
DATASTD-2454

mikeholm-simpat committed:
DATASTD-2454

mikeholm-simpat committed:
DATASTD-2454

mikeholm-simpat committed:
DATASTD-2454

mikeholm-simpat commented:
Rebased as requested on 8/29/25.
Also added additional IdentificationSystemDescriptor xml files for both Candidate and Contact using StaffIdentificationSystemDescriptor as a base. Re-submitting for review.

mikeholm-simpat committed:
DATASTD-2454
